local-cli/eject/eject.js

```diff
@@ -84,6 +129,53 @@ function eject() {
       appName,
       templateOptions
     );
+
+    console.log('Setting up App Icons for Android');
+
+    const ICON_SIZES = [{
+      android_hdpi: {
+        path: 'android/app/src/main/res/mipmap-hdpi/ic_launcher.png',
+        size: 48
+      },
+      android_mdpi: {
+        path: 'android/app/src/main/res/mipmap-mdpi/ic_launcher.png',
+        size: 72
+      },
+      android_xhdpi: {
+        path: 'android/app/src/main/res/mipmap-xhdpi/ic_launcher.png',
+        size: 96
+      },
+      android_xxhdpi: {
+        path: 'android/app/src/main/res/mipmap-xxhdpi/ic_launcher.png',
+        size: 144
+      }
+    }];
+
+    let filePaths = [];
+
+    Object.keys(ICON_SIZES[0]).forEach((prop) => {
+      filePaths.push(ICON_SIZES[0][prop].path);
+    })
+
+    const doesAppIconExist = fs.existsSync(path.resolve(appIconAndroid));
+
+    if( typeof appIconAndroid === 'string' && doesAppIconExist) {
+      filePaths.forEach((path) => {
+        fs.unlink(path, (err) => {
+          if(err) throw err;
+        })
+      })
+    };
+
+    Object.keys(ICON_SIZES[0]).forEach((prop) => {
+      const path = ICON_SIZES[0][prop].path;
+      const size = ICON_SIZES[0][prop].size;
+      sharp(appIconAndroid)
+      .resize(size, size)
+      .toFile(path), function(err) {
```

Comment:
@ndejesus1227 The closing parenthesis from `toFile(path)` is in the wrong place (I'm guessing you meant to pass in the error handler as 2nd argument).  Right now that comma is in the middle of nowhere and invalid JS syntax.